core-common/src/main/java/org/apache/kylin/common/util/HadoopUtil.java

```diff
@@ -64,12 +64,14 @@ public static Configuration getCurrentConfiguration() {
             return conf;
         }
         Configuration conf = hadoopConfig.get();
+        conf.set("fs.hdfs.impl.disable.cache", "true");
```

Comment:
Why it needs to add this parameter here? and will it impact other functions, because there are some place calling this functions?

Reply:
This configuration will not affect other functions, it has been verified,This is done in order to prevent the FileSystem from shutting down prematurely, resulting in abnormal file information acquisition